docs/quickstarts/fre-quickstart.md

```diff
@@ -0,0 +1,176 @@
+
+This quick start provides instructions and screenshots for Excel but you can use a similar pattern to implement a first-run experience in other Office applications where Office Web Add-ins are supported. If you'd like to use a different Office application, see that application's quick start for setup instructions before proceeding. Also note that actual interactions with your selected Office application will differ.
+
+## Prerequisites
+
+[!include[Yeoman generator prerequisites](../includes/quickstart-yo-prerequisites.md)]
+
+## Create the add-in project
+
+[!include[Yeoman generator create project guidance](../includes/yo-office-command-guidance.md)]
+
+- **Choose a project type:** `Office Add-in Task Pane project`
+- **Choose a script type:** `JavaScript`
+- **What do you want to name your add-in?** `My Office Add-in`
+- **Which Office client application would you like to support?** `Excel`
+
+![The Yeoman Generator for Office Add-ins command line interface.](../images/yo-office-excel.png)
+
+After you complete the wizard, the generator creates the project and installs supporting Node components.
+
+[!include[Node.js version 20 warning](../includes/node-20-warning-note.md)]
+
+[!include[Yeoman generator next steps](../includes/yo-office-next-steps.md)]
+
+## Explore the project
+
+[!include[Yeoman generator add-in project components](../includes/yo-task-pane-project-components-js.md)]
+
+## Try it out
+
+1. Navigate to the root folder of the project.
+
+    ```command&nbsp;line
+    cd "My Office Add-in"
+    ```
+
+1. [!include[Start server section](../includes/quickstart-yo-start-server-excel.md)]
+
+    > [!NOTE]
+    > If this is the first time that you have sideloaded an Office add-in on your computer (or the first time in over a month), you're prompted first to delete an old certificate and then to install a new one. Agree to both prompts.
+
+1. A **WebView Stop On Load** prompt appears. Select **OK**.
+
+1. In Excel, choose the **Home** tab, and then choose the **Show Taskpane** button on the ribbon to open the add-in task pane.
+
+    ![The Excel Home menu, with the Show Taskpane button highlighted.](../images/excel-quickstart-addin-3b.png)
+
+1. Select any range of cells in the worksheet.
+
+1. At the bottom of the task pane, choose the **Run** link to set the color of the selected range to yellow.
+
+    ![The add-in task pane open in Excel, with the Run button highlighted in the add-in task pane.](../images/excel-quickstart-addin-3c.png)
+
+## Implement the first-run experience
+
+1. Open the **taskpane.html**. Replace the `<main>` element with the following markup, then save the file.
```

Comment:
Let's add some explanation in each section to what the code is doing. We should teach people why they are doing what they are doing and how it works.